th2_data_services_lwdp/utils/pages.py

```diff
@@ -16,7 +16,11 @@ def __init__(self, data: Dict):  # noqa
         self.name = data["name"]
         self.comment = data["comment"]
         self.start_timestamp = data["started"]
+        self.start_timestamp_tuple = tuple(self.start_timestamp.values())
         self.end_timestamp = data["ended"]
+        self.end_timestamp_tuple = (
+            tuple(self.end_timestamp.values()) if self.end_timestamp else None
+        )
```

Comment:
Do we really need this strange xxx_tuple varibales?

Reply:
data["started"] already have good enough format for ProtobufTimestampConverter

Reply:
I added it because it makes easier to access the timestamps. DatetimeConverter.to_milliseconds accepts either datetime object or tuple. 
I could have added util method to parse it, but I thought it would also be easier for user.

Reply:
ProtobufTimestampConverter will fix it